deps/github.com/openshift/kubernetes/openshift-hack/rebase.sh

```
@@ -98,9 +93,13 @@ git fetch upstream --tags -f
 git remote add openshift git@github.com:openshift/kubernetes.git
 git fetch openshift
 
-#git checkout --track "openshift/$openshift_release"
+git checkout --track "openshift/$openshift_release"
 git pull openshift "$openshift_release"
 
+if [ -z "$(git tag -l "$k8s_tag")" ]; then
```

Shellcheck (0.11.0)

[style] Prefer putting braces around variable references even when not strictly required.

(SC2250)